data/docs-nav.ts

```diff
@@ -23,9 +23,18 @@ const DocsNav: NavItem[] = [
     title: "Step 1: Install",
     path: "/docs/installation",
     children: [
-      { title: "Node.js", path: "/docs/installation/node" },
-      { title: "Docker", path: "/docs/installation/docker" },
-      { title: "Plugins", path: "/docs/installation/plugins" },
+      {
+        title: "Installing Grouparoo with Node.js",
+        path: "/docs/installation/node",
+      },
+      {
+        title: "Installing Grouparoo with Docker",
+        path: "/docs/installation/docker",
+      },
+      {
+        title: "Installing Grouparoo Plugins",
+        path: "/docs/installation/plugins",
+      },
     ],
```

Comment:
I understand the desire to have the titles here match the page titles... but now the nav wraps which isn't great for legibility.  Can these be shorter and not match the page titles?

<img width="298" alt="Screen Shot 2021-07-20 at 2 14 40 PM" src="https://user-images.githubusercontent.com/303226/126396327-8fc13c11-2dce-4ba6-90bd-f01ae0775d32.png">

Reply:
Similar comments below

Reply:
Good point.  I added an optional "linkText" parameter for places we want the link text to be different on those.

Reply:
Good point.  I added an optional 1linkText` property now.